libs/application/templates/children-residence-change/src/fields/ChangeInformation/index.tsx

```diff
@@ -1,37 +1,40 @@
 import React from 'react'
-import { FieldBaseProps, getValueViaPath } from '@island.is/application/core'
+import { FieldBaseProps } from '@island.is/application/core'
 import { Box, Text } from '@island.is/island-ui/core'
+import { CheckboxController } from '@island.is/shared/form-fields'
+import { useLocale } from '@island.is/localization'
 import {
   extractApplicantFromApplication,
   extractParentFromApplication,
   constructParentAddressString,
 } from '../../lib/utils'
-import { CheckboxController } from '@island.is/shared/form-fields'
+import { newResidence } from '../../lib/messages'
+import { DescriptionText } from '../components'
 
 const ChangeInformation = ({ field, application, error }: FieldBaseProps) => {
   const { id, disabled } = field
+  const { formatMessage } = useLocale()
```

Comment:
Consideratoin to use useIntl() instead of this hook since we know the type of the message every time :)